sippy-ng/src/scalejobs/ScaleJobsTable.js

```diff
@@ -0,0 +1,139 @@
+// TODO: separate css for this file?
+import '../jobs/JobTable.css'
+import { Button, Container, Tooltip } from '@material-ui/core'
+import { DataGrid } from '@material-ui/data-grid'
+import { DirectionsRun } from '@material-ui/icons'
+import { Link } from 'react-router-dom'
+import Alert from '@material-ui/lab/Alert'
+import GridToolbar from '../datagrid/GridToolbar'
+import PropTypes from 'prop-types'
+import React, { useEffect } from 'react'
+
+export const getColumns = (config) => {
+  return [
+    {
+      field: 'upstreamJob',
+      headerName: 'upstreamJob',
+      flex: 3.5,
+      renderCell: (params) => {
+        return <div className="job-name">{params.value}</div>
+      },
+    },
+  ]
+}
```

Comment:
This can just be a variable instead of a function  -- I think only the job page uses a function since jobs are special and I needed to fetch the columns from some other pages.